inference-gateway/src/config.rs

```diff
@@ -20,6 +22,15 @@ impl Config {
             server_port: from_env_default("WEBSERVER_PORT", "8080")
                 .parse::<u16>()
                 .unwrap_or(8080),
+            org_validation_enabled: from_env_default("ORG_VALIDATION_ENABLED", "false")
+                .parse()
+                .expect("ORG_VALIDATION_ENABLED must be a boolean"),
```

Comment:
org validation disabled. And there is nothing in place to populate the local validation table.